blog/2024-12-15-openhab-4-3-release.md

```diff
@@ -161,6 +161,63 @@ But not only were new bindings added, some existing bindings also received massi
 - The [JavaScript Scripting automation](/addons/automation/jsscripting) has finally fixed the remaining reported multi-threading issues for UI-bases rules.
 - The [KNX binding](/addons/bindings/knx) provides better support for KNX Secure, allowing to include keyring files and reading KNX Data Secure packets.
 
+### MQTT / Home Assistant
+
+_Cody Cutrer ([@ccutrer](https://github.com/ccutrer))_
```

Comment:
Looking at your last commit, the answer is yes. Authorship goes to the top and typically contains the „role“ of the author, I will add that for you:

```suggestion
_Cody Cutrer ([@ccutrer](https://github.com/ccutrer)), openHAB Maintainer_
```